src/utils/network/session-core.js

```diff
@@ -44,17 +41,23 @@ import {
  * @memberof utils
  * @function setDocAclPermission
  * @param {Session} session - Solid's Session Object {@link Session}
- * @param {string} fileType - Type of document
+ * @param {string} docType - Type of document
  * @param {Access} permissions - The Access object for setting ACL in Solid
- * @param {string} otherPodUsername - Username to other user's Pod or empty string
+ * @param {URL} podUrl - URL of the user's Pod
+ * @param {string} webIdToAssignPermissionsTo - URL of the other user's Pod to give/revoke permissions OR empty string
  * @returns {Promise} Promise - Sets permission for otherPodUsername for given
  * document type, if exists, or null
  */
-export const setDocAclPermission = async (session, fileType, permissions, otherPodUsername) => {
-  const containerUrl = getContainerUrl(session, 'Documents', INTERACTION_TYPES.SELF);
-  const documentUrl = `${containerUrl}${fileType.replace("'", '').replace(' ', '_')}/`;
-  const otherPodUrl = getPodUrl(otherPodUsername);
-  const webId = `${otherPodUrl}profile/card#me`;
+export const setDocAclPermission = async (
+  session,
+  docType,
+  permissions,
+  podUrl,
+  webIdToAssignPermissionsTo
+) => {
+  const containerUrl = `${podUrl}PASS/Documents/`;
+  const documentUrl = `${containerUrl}${docType.replace("'", '').replace(' ', '_')}/`;
+  const webId = webIdToAssignPermissionsTo;
```

Comment:
If we're going to use webIds for the parameters to `setDocAclPermission` and `setDocContainerAclPermission`, we should just pass it into `setDocAclForUser` directly without needing to use `const webId`.